Gaming (#518)
solana-developers/solana-cookbook: docs/gaming/distribution.md

---
title: Distribution
head:
  - - meta
    - name: title
      content: Solana Cookbook | Game Distribution
  - - meta
    - name: og:title
      content: Solana Cookbook | Game Distribution
  - - meta
    - name: description
      content: Different options on how to distribute your game built on Solana
  - - meta
    - name: og:description
      content: Different options on how to distribute your game built on Solana
  - - meta
    - name: og:image
      content: https://solanacookbook.com/cookbook-sharing-card.png
  - - meta
    - name: og:image:alt
      content: Solana splash card
  - - meta
    - name: twitter:card
      content: summary
  - - meta
    - name: twitter:site
      content: "@solanacookbook"
  - - meta
    - name: twitter:image
      content: "https://solanacookbook.com/cookbook-sharing-card.png"
  - - meta
    - name: robots
      content: index,follow,noodp
  - - meta
    - name: googlebot
      content: index,follow
footer: MIT Licensed
---

# Distribution

> Distribution of your game depends highly on the platform you are using of course. With Solana game SDKs you can build for IOS, Android, Web and Native Windows or Mac. Using the Unity SDK you could even connect Nintendo Switch or XBox to Solana theoretically. Many game companies are pivoting to a mobile first approach because there are so many people with mobile phones in the world. Mobile comes with its own complications though, so you should pick what fits best to your game.  

Reviewer: ```suggestion
Distribution of your game depends highly on the platform you are using. With Solana, there are game SDKs you can build for IOS, Android, Web and Native Windows or Mac. Using the Unity SDK you could even connect Nintendo Switch or XBox to Solana theoretically. Many game companies are pivoting to a mobile first approach because there are so many people with mobile phones in the world. Mobile comes with its own complications though, so you should pick what fits best to your game.  
```

> A unique advantage that Solana has compared to other blockchains, is that Solana is selling a crypto native mobile phone called [Saga](https://solanamobile.com/) which comes with a new innovative dApps store which allows you to distribute crypto games without the restrictions on traditional app stores from for examples Google or Apple. 

Reviewer: ```suggestion
Solana has a distinct edge over other blockchain platforms due to its offering of a crypto-native mobile phone, named Saga, that comes equipped with an innovative dApps store. This store enables the distribution of crypto games without the limitations imposed by conventional app stores such as Google or Apple.
```

## Publishing Platforms

Platforms where you can host your games

> | Platform | Description | Link |

Reviewer: Instead of platform | desc | link, could you just hyperlink the platform name with the link?

| --- | --- | --- |

> | Fractal | A game publishing platform that supports Solana and Ethereum. They have an amazing team and are super supportive on bringing on new good games. They also have their own wallet and account handling and there is an SDK for high scores and tournaments  | https://www.fractal.is/ |

Reviewer: `They have an amazing team and are super supportive` - this is an opinion. I'd try to avoid pushing opinions in docs and stick to the facts

| Elixir | Platform for web3 games that also offers a PC launcher | https://elixir.app/ |

> | Vercel | Just host your game yourself. Can be easily setup so that a new version get deployed as soon as you push to your repository.  | https://vercel.com/ |

Reviewer: Is this really considered a game distributor?

Author: Not really only for games, i will change it up.

> | Solana mobile DApp Store | The Solana alternative to the Play and AppStore. A crypto first variant of a dApp store, which is open source free for everyone to use. Applications for the release of the dApp store a open. | https://github.com/solana-mobile/dapp-publishing/blob/main/README.md |

Reviewer: `Applications for the release of the dApp store a open` - what does this mean?

Reviewer: `alternative to the Play and AppStore` -> `alternative to Google Play and the Apple App Store`

> | Ios App Store | The Ios app store has a high reach and is trusted by its customers. The entrance barrier for crypto games is high though. The rules are very strict for everything that tries to circumvent the fees that Apple takes for in app purchases. A soon as an NFT provides benefits for the player for example Apple requires you for example to have them purchased via their in app purchase system.  | https://www.apple.com/de/app-store/ |

Reviewer: ```suggestion
| Apple App Store | The Apple app store has a high reach and is trusted by its customers. The entrance barrier for crypto games is high though. The rules are very strict for everything that tries to circumvent the fees that Apple takes for in app purchases. A soon as an NFT provides benefits for the player for example Apple requires you for example to have them purchased via their in app purchase system.  | https://www.apple.com/de/app-store/ |
```

> | Google Play Store | Google is much more crypto friendly and games with NFTs and wallet deep links for example have a much easier time to be approved for the official play store.  | https://play.google.com/store/games |

Reviewer: ```suggestion
| Google Play Store | Google is much more crypto friendly and games with NFTs and wallet deep links for example have had a track record of being approved for the official play store.  | https://play.google.com/store/games |
```

> | xNFT Backpack | Backpack is a Solana wallet which allows you to release apps as so called xNFTs. They appear in the users wallet as soon as they purchase them and then they can be used. The Unity SDK has a xNFT export and any other web app can be published as xNFT as well. | https://www.backpack.app/ |

Reviewer: ```suggestion
| xNFT Backpack | Backpack is a Solana wallet which allows you to release apps as xNFTs. They appear in the users wallet as soon as they purchase them as applications. The Unity SDK has a xNFT export and any other web app can be published as xNFT as well. | https://www.backpack.app/ |
```
